File: CollaborationBackend/src/main/java/com/collaborate/service/UserService.java
package com.collaborate.service;

import java.util.List;

import com.collaborate.model.Users;

public interface UserService {
	boolean isUserIdValid(String userid);
	boolean isEmailValid(String email);
	public Users login(Users users);
	public boolean createUsers(Users users);
	public boolean isUpdatedEmailValid(String email,String userid);
	public Users getUser(String userid);
	public void editUsers(Users users);

}

File: CollaborationFrontend/Web/Services/userService.js
/**
 * 
 */
myApp.factory('UserService',function($http){
	var userService={}
	var BASE_URL="http://localhost:8081/CollaborationMiddleware"
	userService.createUsers=function(users){
		alert("User Service "+users.userid)
		return $http.post(BASE_URL + "/createUsers",users)
	}
	userService.login=function(users){
		return $http.post(BASE_URL + "/login",users)
	}
	userService.logout=function(){
		return $http.put(BASE_URL + "/logout")
	}
	userService.getUser=function(){
		return $http.get(BASE_URL + "/getUser")
	}
	userService.updateUsers=function(users){
		return $http.put(BASE_URL + "/updateUsers",users)
	}
	return userService;
		
})